app/Database/CourseQueries.hs

```diff
@@ -68,17 +67,17 @@ returnCourse lowerStr = runSqlite databasePath $ do
       Just course -> do
         lecturesList :: [Entity Lecture] <- lectureQuery courseStr
         tutorialsList :: [Entity Tutorial] <- tutorialQuery courseStr
-        let (fallSession, springSession, yearSession) = buildAllSessions lecturesList tutorialsList
-        buildCourse fallSession springSession yearSession (entityVal course)
+        let (fallsession, springsession, yearsession) = buildAllSessions lecturesList tutorialsList
```

Comment:
@AilsaF we want to always use camelCase for variable names. Why don't we just name these "fall" "winter" and "year". Similar comment below.